XF/Pub/View/Member/Find.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace SV\UserMentionsImprovements\XF\Pub\View\Member;
+
+class Find extends XFCP_Find
+{
+	public function renderJson()
+	{
+		$response = parent::renderJson();
+
+		foreach ($this->params['usergroups'] AS $usergroup)
+		{
+			$response['results'][] = [
```

Comment:
This is adding usergroups to the end of the list, They should be added to the start of the list to preserve ordering of preferring usergroup mention's 1st.